eegnb/cli/introprompt.py

```diff
@@ -40,57 +40,61 @@ def intro_prompt():
           f"[10] {boards[10]} \n",
           f"[11] {boards[11]} \n")
 
-    board_idx = int(input('Enter Board Selection:'))
+    board_idx = int(input('Enter Board Selection: '))
     board_selection = board_codes[board_idx]    # Board_codes are the actual names to be passed to the EEG class
     print(f"Selected board {boards[board_idx]} \n")
 
     # Handles wifi shield connectivity selection if an OpenBCI board is being used
     if board_selection in ['cyton', 'cyton_daisy', 'ganglion']:
-        # if the ganglion is being used, will also need the MAC address
-        if board_selection == 'ganglion':
-            print("Please enter the Ganglions MAC address:\n")
-            mac_address = input("MAC address:")
 
         # determine whether board is connected via Wifi or BLE
         print("Please select your connection method:\n"
               "[0] usb dongle \n"
               "[1] wifi shield \n")
-        connect_idx = input("Enter connection method:")
+        connect_idx = int(input("Enter connection method: "))
 
         # add "_wifi" suffix to the end of the board name for brainflow
         if connect_idx == 1:
             board_selection = board_selection + "_wifi"
+            ip_address = input("\nEnter Ganglion+WiFi IP Address: ")
+        else:
+            # if the ganglion is being used, you can enter optional Ganglion mac address
+            if board_selection == 'ganglion':
+                ganglion_mac_address = input("\nGanglion MAC Address (Press Enter to Autoscan): ")
```

Comment:
I don't think this is specific to the ganglion.

```suggestion
            mac_address = input("\nEnter Bluetooth MAC Address (or empty to autoscan): ")
```

Reply:
Only the text description for the prompt was changed.

Reply:
Yeah I noticed, just saw something that could be simplified and suggested it.

Reply:
@ErikBjare the ganglion is the only device in brainflow which uses a MAC address.

Reply:
The most important change right here is that I have tested Ganglion+BLED112 without a mac address on my Mac. Works great even leaving this blank (thanks to BrainFlow)!